docs/lsp/howto/migrate-to-v1.rst

```diff
@@ -0,0 +1,76 @@
+.. _lsp-v1-migration:
+
+How To Migrate to v1
+====================
+
+.. note::
+
+   Since v1.0 is still under development, some features from ``0.x`` may still be missing.
+   Also the details in this guide may change over time.
+
+This guide covers the breaking changes between the ``v0.x`` and ``v1.x`` versions of the language server and how to adapt to them.
+
+.. highlight:: console
+
+Installation Changes
+--------------------
+
+Previously, you would have had to install ``esbonio`` as a development dependency for every project you wished to use it in.
```

Comment:
Has also been working fine when installed globally (to `/usr/lib/python3.xx/site-packages`), as done in https://archlinux.org/packages/extra/any/esbonio/.

Reply:
Oh, maybe this was when using the VS Code extension?

Reply:
Thanks for the feedback!

> Has also been working fine when installed globally 

The guide should probably have said something like the "recommended approach was..."

The key thing is that the `0.x` version needs to be installed alongside Sphinx and all of your docs dependencies in order to work properly. In the new version this restriction is going to go away :)